Test_oM/Attributes/PathAttribute.cs

```diff
@@ -36,6 +36,11 @@ public class PathAttribute : ConditionAttribute, IImmutable
     {
         public virtual Regex Pattern { get; }
 
+        public PathAttribute(Regex pattern, bool expect = true, object typeId = null) : base(expect)
```

Comment:
Can I double check, what are we doing (or planning to do) with `typeId` - I can't see it being used in this constructor or passed to the base like `expect` is? (That said I haven't run through everyone on this and the other PR yet, just doing a quick glance between tasks, but leaving a comment here in case I forget later/or an answer can be provided before I get that far)